azure-pipelines.yml

```diff
@@ -1,6 +1,15 @@
 variables:
   PYTHON: 'python2.7'
 
+trigger:
+  batch: true
+  branches:
+    include:
+      - development
```

Comment:
Given we don't use Azure Pipelines for generating builds, I think this list is fine. But something to keep an eye on in the future if we decide to simplify our CI infrastructure.

Reply:
yep, agreed!